docs/README.md

```diff
@@ -36,10 +36,10 @@ These docs are for
 
 ## Philosophy
 
-Like most frameworks, luma.gl has been developed with a certain philosophy in mind, reflecting things its designers feel to be important to their work.
+luma.gl's core philosophy is to expose the WebGL 2 API to developers, while providing fallbacks to WebGL 1 when necessary. The core use case for luma.gl is visualization of geospatial datasets, but its design is generic enough for more general usage. Key aspects of that philosphy are:
```

Comment:
I would say "very large datasets" and maybe leave out geospatial (that is kind of deck's thing).